src/Plugin/resource/DataProvider/DataProviderEntity.php

```diff
@@ -527,13 +527,13 @@ protected function getQueryForList() {
     try {
       $this->queryForListSort($query);
     }
-    catch (ServerConfigurationException $e) {
+    catch (BadRequestException $e) {
```

Comment:
Can you elaborate a bit more on why this is needed?

Reply:
No ServerConfigurationException is thrown inside queryForListSort(), if you look at the method body, you'll see that only BadRequestException are thrown.